docs/pages/cloud/introduction.mdx

```diff
@@ -1,16 +1,33 @@
 ---
 title: Cloud
 description: Teleport as a service
-h1: Teleport cloud
+h1: Teleport Cloud
 videoBanner: 1jhKOtBinm4
 ---
 
-We run Teleport Cloud as hosted, managed Teleport as a service.
-Connect your nodes, web applications, kubernetes clusters and databases.
+# Teleport Cloud
+
+Teleport Cloud provides a SOC 2 certified Highly Available (HA) hosted Teleport Cluster service
+for your organization.  The same software used for Self-Hosted is managed by Teleport and deployed
+in a secure, scalable tenant dedicated to your organization.
+
 Sign up [here](https://goteleport.com/get-started/).
 
+Key parts of the Teleport Cloud service:
+
+- Removes the need to host the Proxy/Auth services within your infrastructure.
+- Automatically scales as you add resources and usage with no additional cost.
+- No open ports required in your infrastructure for Teleport agents.
+- Low-latency design for world-wide efficiency whereever your resources are.
+- Automatically upgraded to get the latest features as they are released. 
+
 ## Next Steps
 
```

Comment:
This section might look cleaner without the first level of bullets. We could turn the first and second bullets into regular paragraphs, then turn the second-level bulleted list into a first-level bulleted list. Then we could make the final two bullets a single paragraph.